appveyor.yml

```diff
@@ -91,4 +89,6 @@ install:
 build: false
 
 test_script:
+  - "conda info -a"
+  - conda list
   - "%CMD_IN_ENV% %TEST_CMD%"
```

Comment:
Let's go ahead and get rid of the `%CMD_IN_ENV%` thing as long as we are cleaning up. That magic isn't necessary for python 3.5 or later.

```suggestion
```

Reply:
This suggestion shows up as removing the whole line (right?), but it sounds like @mwcraig is saying only remove the `%CMD_IN_ENV%` part. How should I change this?

Reply:
oh, you're right. That case you cannot accept the suggestion, but just  do the edit yourself.